debian/ooniprobe-cli.timer

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=OONI CLI Probe
+Requires=ooniprobe-cli.service
+
+[Timer]
+Unit=ooniprobe-cli.service
+OnCalendar=daily
+RandomizedDelaySec=24h
```

Comment:
Are we sure we would like to do 24h intervals? I was thinking it might be better to do it more frequently like every 6 hours so we can cover the day better.

Moreover, do we maybe want to add a random delay somewhere in here?

Reply:
My understanding is that `RandomizedDelaySec` is already random. I have a feeling we should start with a very conservative value here for packages and then see what happens and choose how to proceed?

Reply:
> My understanding is that RandomizedDelaySec is already random [...]

This is literally my understanding from reading the man page but I have not tested it. Though, I assume that `OnCalendar` plus `RandomizedDelaySec` gives us a single run per day w/ random delay uniform in [0..24h].

Reply:
According to the logs it's randomized as described.

Reply:
Thanks!